scripts/update/2z-job-loss-by-industry-by-state-update.R

```diff
@@ -24,7 +24,11 @@ state_claims <- read_excel(str_glue("data/raw-data/small/{filename}"),
                            sheet = "Sheet1")
 
 # Read in national industry estimates
-national_industry_est <- read_csv("data/processed-data/job_change_all_states_most_recent.csv")
+if (is_bls){
+  national_industry_est <- read_csv("data/processed-data/job_change_bls_most_recent.csv")
+} else {
+  national_industry_est <- read_csv("data/processed-data/job_change_all_states_most_recent.csv")
+}
 
```

Comment:
Just making sure I'm understanding exactly how we're using the BLS data in script 2b, but it looks like from the raw CES data we are getting:
1) The net jobs lost in the latest month for each industry
2) The net jobs lost in the reference month (Feb) for each industry
3) Calculating the percentage difference in net jobs lost in latest month vs Feb.